src/createContext.ts

```diff
@@ -32,21 +32,21 @@ export const createEmptyContext = <T>(
   runtime: createEmptyRuntime()
 })
 
-export const ensureGlobalEnvironmentExist = (context: Context) => {
+export const ensureGlobalFrameExist = (context: Context) => {
```

Comment:
Why is this being renamed to frame?

Reply:
Yes, this should be ensureGlobalEnvironmentExist.

Another question: Why does a Context need multiple Environments?